app/femr/ui/controllers/TriageController.java

```diff
@@ -125,6 +132,13 @@ public Result indexPopulatedGet(int patientId) {
         viewModelGet.setPatient(patient);
         viewModelGet.setVitalNames(vitalServiceResponse.getResponseObject());
         viewModelGet.setPossibleAgeClassifications(patientAgeClassificationsResponse.getResponseObject());
+        //Patient has an open encounter for medical
+        if(patientEncounter.getMedicalDateOfVisit() == null){
```

Comment:
AJ,

An encounter also has a time limit of 2 days set to close if the medical date of visit is null. Try 'patientEncounter.getIsClosed()' instead of 'patientEncounter.getMedicalDateOfVisit()'. :)

The logic for a closed encounter can be found here:

https://github.com/FEMR/femr/blob/8ce6459eb8828d95855ec72e635cc56eaf462121/app/femr/business/helpers/LogicDoer.java#L98

-Kevin